src/TortoiseProc/CommitDlg.h

```diff
@@ -228,13 +268,24 @@ class CCommitDlg : public CResizableStandAloneDialog, public CSciEditContextMenu
 	HACCEL				m_hAccel;
 	bool				RestoreFiles(bool doNotAsk = false, bool allowCancel = true);
 
+	// State of the links "Partial Staging>>" and "Partial Unstaging>>" shown on the commit window
+	enum PartialStagingDisplayState : unsigned char

```

Comment:
I still wonder why we use a bitset here as one needs to be enabled, right?

Reply:
Both can be set at the same time (see the constructor of `CCommitDlg`), but never unset at the same time.

When either one is set, its corresponding link displays "Partial (un)staging". When either one is unset, its corresponding link displays "Hide (un)staging". So, if both are set, the View Patch window is closed and the user can click either one of them to display the partial staging or unstaging window.

Reply:
Yeah, I see.